index.html

```diff
@@ -0,0 +1,79 @@
+    <title>Document</title>
+</head>
+<body>
+    <div id="header">
+      <h1 id="title">Survey</h1>
+      <p id="description">Survey about favorite beverage</p>
+    </div>
+    <form id="survey-form">
+        <div class="question-group">
+            <label class="label-title" id="name-label" for="name">Name</label>
+            <input id="name" type="text" placeholder="Enter your name" required>
+        </div>
+        <div class="question-group">
+            <label class="label-title" id="email-label" for="email">Email</label>
+            <input id="email" type="email" placeholder="Enter your email" required>
+        </div>
+        <div class="question-group">
+            <label class="label-title" id="number-label" for="number">Age</label>
+            <input id="number" type="number" placeholder="Enter your age (optional)" min="10" max="80">
+        </div>
+
+        <div class="question-group">
+            <label class="label-title" id="beverage-label" for="beverage">What it your favorite beverage?</label>
+            <select id="dropdown" name="beverage" required>
+              <option disabled selected value>Select current beverage</option>
+              <option value="coffee">Coffee</option>
+              <option value="tee">Tee</option>
+              <option value="soda">Coke or Soda</option>
+              <option value="water">Water</option>
+              <option value="other">Other</option>
+            </select>
+          </div>
+          <div id="button group" class="question-group">
+            <label class="label-title" id="beverage-label" for="beverage">How often you drink your favorite beverage in a day?</label>
+              <input name="a-lot" value="a-lot" type="radio" class="input-radio" checked>A lot</label>
+            <label>
+              <input name="many-times" value="many-times" type="radio" class="input-radio">Many times</label>
+            <label>
+                <input name="user-recommend" value="sometimes" type="radio" class="input-radio">Sometimes</label>
+          </div>
+
+          <div id="checkbox-group" class="question-group">
+            <label class="label-title" id="beverage-label" for="beverage">Where do you drink it mostly?</label>  
+            <label>
+                <input name="where-drink" value="outdoor" type="checkbox" class="input-checkbox">Outdoor</label>
+            <label>
+              <input name="where-drink" value="at-home" type="checkbox" class="input-checkbox">At home</label>
+            <label>
+                <input name="where-drink" value="at-restaurant" type="checkbox" class="input-checkbox">At restaurants</label>
+            <label>
+                <input name="where-drink" value="other" type="checkbox" class="input-checkbox">Other</label>
+          </div>
+
+          <div id="text-box" class="question-group">
+            <p>Is there anything else you want to tell us about your favorite beverage?</p>
```

Comment:
✅I think that instead of using a <p> tag, it might be more appropriate to use an <h2> element in this case. This can help ensure that the structure and hierarchy of your content are clear and consistent, which can greatly enhance the readability and user experience of your website.